compiler/rustc_feature/src/unstable.rs

```diff
@@ -213,6 +213,8 @@ declare_features! (
     (internal, negative_bounds, "1.71.0", None),
     /// Allows using `#[omit_gdb_pretty_printer_section]`.
     (internal, omit_gdb_pretty_printer_section, "1.5.0", None),
+    /// Set the maximum pattern complexity allowed (not limited by default).
+    (internal, pattern_complexity, "1.78.0", None),
```

Comment:
You have to use `CURRENT_RUSTC_VERSION` instead of `"1.78.0"` to make tidy happy

Reply:
Ah indeed, thanks!